srcpkgs/lilyterm/template

```diff
@@ -1,15 +1,50 @@
 # Template file for 'lilyterm'
 pkgname=lilyterm
-version=0.9.9.4
-revision=2
-build_style="gnu-configure"
-hostmakedepends="pkg-config which"
-makedepends="vte-devel gtk+-devel"
+_githash=faf1254f46049edfb1fd6e9191e78b1b23b9c51d
+version=0.9.9.4+20190725
+revision=1
+wrksrc=LilyTerm-$_githash
```

Comment:
Unfortunate, but I think it's worth it being able to drop vte.

Reply:
Other distros also ships some git version, too.